third_party/move/move-vm/runtime/src/storage/dummy.rs

```diff
@@ -1,103 +1,7 @@
 // Copyright (c) The Move Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::{
-    loader::{Module, Script},
-    storage::{code_storage::CodeStorage, module_storage::ModuleStorage},
-};
-use bytes::Bytes;
-use move_binary_format::{
-    errors::{Location, PartialVMError, VMResult},
-    file_format::CompiledScript,
-    CompiledModule,
-};
-use move_core_types::{
-    account_address::AccountAddress, identifier::IdentStr, metadata::Metadata,
-    vm_status::StatusCode,
-};
-use std::sync::Arc;
-
-/// An error which is returned in case unimplemented code is reached. This is just a safety
-/// precaution to avoid panics in case we forget some gating.
-macro_rules! unexpected_unimplemented_error {
-    () => {
-        Err(
-            PartialVMError::new(StatusCode::UNKNOWN_INVARIANT_VIOLATION_ERROR)
-                .with_message("New loader and code cache are not yet implemented".to_string())
-                .finish(Location::Undefined),
-        )
-    };
-}
-
 // TODO(loader_v2): Temporary infra to enable loader V2 to test & run things e2e locally, remove.
 pub fn use_loader_v2_based_on_env() -> bool {
```

Comment:
hm, let's also ship this somewhere (where it's used) and get rid of the file?

Reply:
Did this in one of the later PRs: #14495